Some weights of the model checkpoint at allenai/longformer-base-4096 were not used when initializing LongformerModel: ['lm_head.dense.bias', 'lm_head.decoder.weight', 'lm_head.layer_norm.weight', 'lm_head.dense.weight', 'lm_head.layer_norm.bias', 'lm_head.bias']
- This IS expected if you are initializing LongformerModel from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPreTraining model).
- This IS NOT expected if you are initializing LongformerModel from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model).
Some weights of the model checkpoint at emilyalsentzer/Bio_ClinicalBERT were not used when initializing BertModel: ['cls.predictions.decoder.weight', 'cls.seq_relationship.bias', 'cls.predictions.transform.dense.bias', 'cls.predictions.transform.dense.weight', 'cls.predictions.transform.LayerNorm.weight', 'cls.predictions.bias', 'cls.seq_relationship.weight', 'cls.predictions.transform.LayerNorm.bias']
- This IS expected if you are initializing BertModel from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPreTraining model).
- This IS NOT expected if you are initializing BertModel from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model).
ehr loaded
cxr loaded
rr loaded
==> training
running for fusion_type fused_ehr
0
starting val epoch 0
Traceback (most recent call last):
  File "fusion_main.py", line 137, in <module>
    trainer.train()
  File "/scratch/baj321/MedFuse/trainers/DHF_trainer.py", line 543, in train
    ret = self.validate(self.val_dl)
  File "/scratch/baj321/MedFuse/trainers/DHF_trainer.py", line 465, in validate
    sorted_modalities = [modalities_list[idx] for idx in sorted_indices.cpu().numpy()]
  File "/scratch/baj321/MedFuse/trainers/DHF_trainer.py", line 465, in <listcomp>
    sorted_modalities = [modalities_list[idx] for idx in sorted_indices.cpu().numpy()]
TypeError: only integer scalar arrays can be converted to a scalar index